types/sinon/index.d.ts

```diff
@@ -392,6 +393,13 @@ declare namespace Sinon {
          * Since sinon@2.0.0
          */
         resolves(value?: any): SinonStub;
+        /**
+         * Causes the stub to return the argument at the provided index.
+         * stub.returnsArg(0); causes the stub to return the first argument.
```

Comment:
Copy/paste error? I think this should be "`stub.resolvesArg(0);` causes the stub to resolve..." and "resolved" instead of "returned" below.

Also according to the sinon docs this function is newer than `returnsArg` and always throws `TypeError` if the index is invalid.

Thanks for adding the doc comment though :)